src/styles/index.css

```diff
@@ -0,0 +1,127 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.card {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  box-sizing: border-box;
+  height: 408px;
```

Comment:
[checklist:](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules#css) Don't fix container size (if there is no such a requirement)
 add padding bottom instead of height